website/src/theme/BlogPostItem/BlogPostBox.js

```diff
@@ -87,7 +87,10 @@ export default function BlogPostBox({metadata = {}, assets, frontMatter}) {
                     )}
                     <TitleHeading className={styles.blogPostTitle} itemProp="headline">
                         {location.pathname.startsWith('/blog') ?
-                                <Link itemProp="url" to={permalink}>
+                                <Link itemProp="url" to={permalink} onClick={(e) => {
+                                        e.preventDefault();
+                                        window.open(permalink, '_blank', 'noopener,noreferrer');
+                                    }}>
```

Comment:
this is a workaround. how did "Video Guides" achieve it? can it be done the same way with no extra code maintenance?

<img width="172" height="149" alt="Screenshot 2025-09-12 at 10 13 08 PM" src="https://github.com/user-attachments/assets/6765285c-ff6d-4f05-9047-a13469e0c3f9" />

Reply:
@rangareddy that's not what i meant. i mean this fix is a workaround. Video guides achieved opening a tab without this change?